mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,10 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Smuggler:
```

Comment:
Smugglers used Ordos units and structures and had their own Starport.
Mercenaries used Ordos units and structures and had their own Heavy Factory.
Fremen used Atreides units and structures.
House Corrino (the Imperial family) used Harkonnen units and structures and had their own Palace.

Might as well leave those for a separate PR.